scripts/gulpfiles/build_tasks.js

```diff
@@ -216,7 +216,7 @@ var JSCOMP_ERROR = [
   'unusedPrivateMembers',
   'uselessCode',
   'untranspilableFeatures',
-  'visibility'
+  // 'visibility'
```

Comment:
I think it would be good to add a comment about why this is commented out.  (Perhaps refer to the jsdoc for `flattenCorePaths` for background, and then just mention about temporarily disabling checking for `@package` due to ES by-path imports.)